x/denom/keeper/createdenom.go

```diff
@@ -7,10 +7,20 @@ import (
 	"github.com/Team-Kujira/core/x/denom/types"
 )
 
+func (k Keeper) GetCreationFee(ctx sdk.Context, creatorAddr string) sdk.Coins {
+	params := k.GetParams(ctx)
+	for _, acc := range params.NoFeeAccounts {
```

Comment:
This iteration looks like it could get expensive? Any way to have `NoFeeAccounts` to be a map?